README.md

```diff
@@ -132,21 +132,32 @@ how to operate in advance use cases, use case by use case
 explanation of all available options/configurations that can be used
 
 # Architecture
-![Container architecture](/docs/assets/container-architecture.jpg "Container architecture")
+
+There is mainly 2 stack you can use 1. rest.bitcoin.com techstack 2. Chris Troutner JS tech stack
```

Comment:
Is it possible to run the 2 stacks at once?

Reply:
Not at the moment